src/constants/moveHistoryEventTemplate.js

```diff
@@ -450,11 +451,19 @@ export const undefinedEvent = buildMoveHistoryEventTemplate({
   eventName: '*',
   tableName: '*',
   detailsType: detailsTypes.PLAIN_TEXT,
-  getEventNameDisplay: () => {
-    return 'Undefined event type';
+  getEventNameDisplay: (historyRecord) => {
+    switch (historyRecord.action) {
+      case dbActions.INSERT:
+        return `Created new item in ${formatTableName(historyRecord.tableName)}`;
+      case dbActions.DELETE:
+        return `Deleted item in ${formatTableName(historyRecord.tableName)}`;
+      case dbActions.UPDATE:
+      default:
+        return `Updated item in ${formatTableName(historyRecord.tableName)}`;
+    }
```

Comment:
```suggestion
  getEventNameDisplay: ({ action, tableName }) => {
    switch (action) {
      case dbActions.INSERT:
        return `Created new item in ${formatTableName(tableName)}`;
      case dbActions.DELETE:
        return `Deleted item in ${formatTableName(tableName)}`;
      case dbActions.UPDATE:
      default:
        return `Updated item in ${formatTableName(tableName)}`;
    }
```

I was discussing this with @YanZ777 yesterday while pairing, and the standard for the codebase is all over the place on this topic. But do you think it's better for maintainability to use ES6 de-structuring as arguments to save on having to access `obj.prop` each time? I think I've made the mistake of refactoring functions in the opposite direction of my suggestion above. @michaelvaldes-truss do you have an opinion on this?

Reply:
We can do this as a cleanup task. There's no user-facing changes for this so we don't need to hold up this PR which contains UAT testable things.